slides/src/content/components.md

```diff
@@ -312,6 +312,75 @@ class ChildComponent {}
 
 ---
 
+## Template Projection (2/3)
+
+- Multiple `<ng-content>` tags can be used in a template by using the `select` attribute
+- In the template, we can use an HTML tag, say, `<header>` to specify the position of projected content to the `ng-content` with `select="header"`
+
+```ts
+@Component({
+  selector: 'rio-app',
+  template: `<rio-child>
+              <header>
+               <p>This is my header content</p>
+              </header>
+              <footer>
+                <p>This is my footer content</p>
+              </footer>
+             </rio-child>`
+})
+class AppComponent {}
+```
+
+```ts
+@Component({
+  selector: 'rio-child',
+  template: `<h4>Child Component</h4>
+             <ng-content select="header"></ng-content>
+             <ng-content select="footer"></ng-content>`
+})
+class ChildComponent {}
+```
+
+[View Example](https://plnkr.co/edit/kwD3iKLU8mELAoHj2fBv?p=preview)
+
+---
+
+## Template Projection (3/3)
+
+- Besides tags, another option for specifying which `ng-content` tag to use is CSS classes
+- This can be done by setting the value of the `select` attribute to a class selector such as `.header-content`
+- Wrap the desired content in a `<div>` with the matching CSS class to specify content position
+
+```ts
+@Component({
+  selector: 'rio-app',
+  template: `<rio-child>
+              <div class="header-content">
+               <p>This is my header content</p>
+              </div>
+              <div class="footer-content">
+                <p>This is my footer content</p>
+              </div>
+             </rio-child>`
+})
+class AppComponent {}
+```
+
+```ts
+@Component({
+  selector: 'rio-child',
+  template: `<h4>Child Component</h4>
+             <ng-content select=".header-content"></ng-content>
+             <ng-content select=".footer-content"></ng-content>`
+})
+class ChildComponent {}
+```
+
+[View Example](https://plnkr.co/edit/YQTUGbOxhxZ41iKKpTbV?p=preview)
```

Comment:
Same clipping problem as the other slide

<img width="1440" alt="screen shot 2017-02-01 at 5 14 48 pm" src="https://cloud.githubusercontent.com/assets/2279046/22528595/ffc1fb5c-e8a1-11e6-9395-7be63517a7cf.png">